pyanaconda/ui/gui/spokes/subscription.glade

```diff
@@ -725,10 +725,10 @@
                                 <property name="can_focus">False</property>
                                 <property name="spacing">4</property>
                                 <child>
-                                  <object class="GtkLabel" id="properly_subscribed_label">
+                                  <object class="GtkLabel" id="subscription_status_label">
                                     <property name="visible">True</property>
                                     <property name="can_focus">False</property>
-                                    <property name="label" translatable="yes">The system has been properly subscribed</property>
+                                    <property name="label">""</property>
```

Comment:
Should this be untranslatable?

Reply:
It's now set dynamically at runtime from Python code:
```
        if self._subscription_module.IsRegisteredToSatellite:
            self._subscription_status_label.set_text(
                _("The system is registered to a Satellite instance.")
            )
        else:
            self._subscription_status_label.set_text(
                _("The system is registered.")
            )
```
I guess that should work fine ?

Reply:
Ah, right, that makes sense, sorry.